Source/FieldSolver/FiniteDifferenceSolver/MacroscopicProperties/MacroscopicProperties.H

```diff
@@ -263,20 +271,22 @@ public:
      /** Multifab storing spatially varying saturation magnetization */
      std::unique_ptr<amrex::MultiFab> m_mag_Ms_mf;
      /** Multifab storing spatially varying Gilbert damping */
```

Comment:
Could we also change the comment above to read 
`Multifab storing spatially varying Gilbert damping on three faces`

Reply:
fixed in d041126e7b593084f342175a65a7e8cb84561cbc